### ISSUE TITLE
WCAG: Iframe needs title

### ISSUE THREAD

**Issue by tmkarlsen** [cerpus/Edlib#1559]
Missing title attribute on iframe

WCAG-connection
WCAG 2.0 level AA:
1.3.1 Info and relationship (A)

From the current NDLA Facade iframe:
<iframe loading="lazy" width="773" height="579.75" allowfullscreen="allowfullscreen" src="https://h5p.ndla.no/resource/a7bc8530-fbdc-4dc0-b0d5-9513064c7c96?locale=nb-no&amp;cssUrl=https%3A%2F%2Fndla.no%2Fstatic%2Fh5p-custom-css.css" data-ratio="0.75"...

- Could the iframe display the resource title?
- Look into if there are iframes within H5P that are missing title/describing info like e.g., video

**Comment by tmkarlsen** [comment]
Estimate 5,5 h (PERT)
Standard deviation 2,2 h

**Comment by tmkarlsen** [comment]
Knowit says:
For H5P så forsøker vi å hente tittel og sette i embed, men det fungerer kun dersom h5p har satt en lisens. Det er fordi endepunktet /v1/resource/{resource}/copyright kun returnerer dersom lisens er satt. Dersom dette endepunktet hadde blitt endra til å alltid returnere noko, men då med tomt lisensfelt dersom ikkje satt, vil vi kunne alltid vise title på disse iframene.
